pkgs/top-level/python-packages.nix

```diff
@@ -152,6 +152,32 @@ in {
     };
   };
 
+  ansicolor = buildPythonPackage rec {
+    name = "ansicolor-${version}";
+    version = "0.2.4";
+
+    src = pkgs.fetchurl{
```

Comment:
Better to replace with `fetchFromGitHub`. 

```nix
fetchFromGitHub {
  owner = "numerodix";
  repo = "ansicolor";
  rev = version;
  sha256 = "...";
}
```

Also, the package is available on pypi, so it might be preferred.